website/client/js/controllers/rootCtrl.js

```diff
@@ -185,23 +185,73 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
           $rootScope.charts[id] = (history.length == 0) ? false : !$rootScope.charts[id];
           if (task && task._editing) task._editing = false;
       }
+
+      if($rootScope.charts[id]) {
+        var handleResize = _.debounce(function() {
+          drawChart(id, data);
+        }, 300);
+
+        $(window).resize(handleResize);
```

Comment:
This is adding a new event listener to the window resize every time the chart is toggled. Meaning if you toggle back and forth a few times, you add a new listener each time and the old listeners do not get cleaned up. This will be a performance issue.